File: src/main/kotlin/aris/thesis/theratricalplaysapi/repositories/OrganizerRepository.java
package aris.thesis.theratricalplaysapi.repositories;

import aris.thesis.theratricalplaysapi.entities.Organizer;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

public interface OrganizerRepository extends JpaRepository<Organizer, Integer>, JpaSpecificationExecutor<Organizer> {

}

File: src/main/kotlin/aris/thesis/theratricalplaysapi/repositories/OrganizerRepository.kt
package aris.thesis.theratricalplaysapi.repositories

import aris.thesis.theratricalplaysapi.entities.Organizer
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor

interface OrganizerRepository : JpaRepository<Organizer?, Int?>, JpaSpecificationExecutor<Organizer?>